momento-sdk/src/main/java/PreviewStorageClientDriver.java

```diff
@@ -0,0 +1,89 @@
+import momento.sdk.IPreviewStorageClient;
+import momento.sdk.PreviewStorageClient;
+import momento.sdk.exceptions.ClientSdkException;
+import momento.sdk.responses.storage.control.CreateStoreResponse;
+import momento.sdk.responses.storage.control.ListStoresResponse;
+import momento.sdk.responses.storage.data.GetResponse;
+
+public class PreviewStorageClientDriver {
+  public static void main(String[] args) {
+    // Instantiating the client will be largely the same as compared to cache/topics.
+    IPreviewStorageClient client = new PreviewStorageClient();
```

Comment:
don't we need config/creds here?

Reply:
For now I've written requests, responses, the interface, and integration tests. I was going to save connecting and the credentials for when the service is live. Shall I do that now?

Reply:
Added in [14c4367](https://github.com/momentohq/client-sdk-java/pull/358/commits/14c4367eef290a467d15bd096c191323c2d932c1)